src/main/java/org/jboss/logmanager/ExtLogRecord.java

```diff
@@ -615,4 +619,15 @@ public void setResourceBundle(final ResourceBundle bundle) {
     public void setResourceBundleName(final String name) {
         super.setResourceBundleName(name);
     }
+
+    /**
+     * Set the marker for this event. Markers are used mostly by SLF4J and Log4j.
+     */
+    public void setMarker(Serializable marker) {
```

Comment:
Markers should not, IMO, be constrained to be `Serializable` (for the same reason that MDC values are not so constrained).